.github/workflows/aks-addon-tests.yml

```diff
@@ -23,7 +23,7 @@ env:
 
 jobs:
   export-registry:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Can you please add `environment: e2e-tests` so we can update the `pull_request_target` to `pull_request`?